target/linux/generic/hack-4.14/302-powerpc-Enable-kernel-XZ-compression-option-on-BOOK3.patch

```diff
@@ -0,0 +1,37 @@
+From 26064848efbca49c643d1237dc1f8215515d52ee Mon Sep 17 00:00:00 2001
+From: Aaro Koskinen <aaro.koskinen@iki.fi>
+Date: Tue, 19 Jun 2018 23:52:30 +0300
+Subject: [PATCH] powerpc: Enable kernel XZ compression option on BOOK3S_32
+
+Enable kernel XZ compression option on BOOK3S_32. Tested on G4
+PowerBook.
+
+Signed-off-by: Aaro Koskinen <aaro.koskinen@iki.fi>
+[mpe: Use one select under the PPC symbol guarded by if PPC_BOOK3S]
+Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
+[added PPC_85xx too]
+---
+
+--- a/arch/powerpc/Kconfig
++++ b/arch/powerpc/Kconfig
+@@ -199,6 +199,7 @@ config PPC
+ 	select HAVE_IOREMAP_PROT
+ 	select HAVE_IRQ_EXIT_ON_IRQ_STACK
+ 	select HAVE_KERNEL_GZIP
++	select HAVE_KERNEL_XZ			if PPC_BOOK3S || PPC_85xx
```

Comment:
I guess at some point this should be sent upstream. to linux-ppc. 
But of course not like this hacked patch. Do you want to do the honors?
You could use https://github.com/linuxppc/linux/commit/423bfc69d7f491c47fc35921f7d460be4094d555 as a template and just replace the 44x with PPC_85xx

(No, I really don't want to do as I don't have that particular hardware.)

Reply:
I sent patch to upstream. I will change hack patches to pending patches.